src/main/java/org/zalando/nakadi/repository/db/SubscriptionDbRepository.java

```diff
@@ -148,7 +148,11 @@ public void deleteSubscription(final String id)
         }
 
         queryBuilder.append(" ORDER BY s_subscription_object->>'created_at' DESC LIMIT ? OFFSET ? ");
-        params.add(limit);
```

Comment:
the change is not clear, there is a default value passed of 20 and there is a check in SubscriptionService

Reply:
ah, I see what you do down there

Reply:
if response from DB will be too big, it will Nakadi, for me it makes sense to use pagination

Reply:
Fixed in 14d8fd6 and 89b80aa